modules/help_editor/ajax/process.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * This file contains code for editing help content.
+ * It holds the processing of adding/updating content
+ * for all modules.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Zaliqa Rosli <zaliqa.rosli@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+namespace LORIS\help_editor;
+
+require_once 'Database.class.inc';
+require_once 'helpfile.class.inc';
+
+$user = \User::singleton();
+if (!$user->hasPermission('context_help')) {
+    header("HTTP/1.1 403 Forbidden");
+    exit(
+        "You do not have the correct permissions for this operation."
+    );
+}
+
+$DB = \Database::singleton();
+//Get the default values
+// Sanitize user input
```

Comment:
This doesn't actually do any sanitizing as far as I can see! This likely isn't a problem because this is taken care of elsewhere in the code base, but this comment should be removed as it's misleading.